### PR TITLE
fix mock session validation module + simple execution module

### PR DESCRIPTION
**Pull request by livingrockrises**

### CODE REVIEW

File: contracts/smart-account/modules/SimpleExeuctionModule.sol
Comment (filmakarov) [inline contracts/smart-account/modules/SimpleExeuctionModule.sol:1]: Typo in the file name

**Review by filmakarov — APPROVED** [review]

**Comment by filmakarov** [comment]
Hey @Chirag018 do we still need this PR?

**Comment by filmakarov** [comment]
Hey @livingrockrises do we still need this PR?

**Comment by livingrockrises** [comment]
No, it's been in draft and I will close it. for delegate call module / simple execution module I will open a new one.

**Pull request closed without merging**